src/components/Header/groups/SwapNavGroup.tsx

```diff
@@ -120,9 +120,8 @@ const SwapNavGroup = () => {
                   <StyledBridgeIcon height={15} />
                 </IconWrapper>
                 <Flex alignItems={'center'} sx={{ flex: 1 }} justifyContent={'space-between'}>
-                  <Text>
-                    <Trans>Bridge</Trans>
-                  </Text>
+                  <Trans>Bridge</Trans>
```

Comment:
can you help explain why remove <Text>

Reply:
redundant div tag.

Reply:
<Text as='span'> can help? Just curious, not required to add it back.